practice-one/src/styles/common/buttons.css

```diff
@@ -0,0 +1,26 @@
+.btn {
+  display: block;
+  margin: 0 auto;
+  color: #FFFFFF;
```

Comment:
You should use hex color codes and lowercase text for color values. This is the convention.
`color: #FFFFFF;` => `color: #fff`